modulefiles/build.hera.intel.lua

```diff
@@ -41,8 +41,8 @@ load(pathJoin("w3emc", w3emc_ver))
 sigio_ver=os.getenv("sigio_ver") or "2.3.2"
 load(pathJoin("sigio", sigio_ver))
 
-zlib_ver=os.getenv("zlib_ver") or "1.2.13"
-load(pathJoin("zlib", zlib_ver))
+--zlib_ver=os.getenv("zlib_ver") or "1.2.13"
+--load(pathJoin("zlib", zlib_ver))
```

Comment:
Was zlib causing a problem or is it not used?

Reply:
I removed the module load of zlib for the intelllvm build. Everything built fine. Apparently, zlib is automatically loaded from some other module. I only tried this test on Hera.